h11/_headers.py

```diff
@@ -62,6 +68,28 @@
 _field_value_re = re.compile(field_value.encode("ascii"))
 
 
+class Headers(Sequence):
+    def __init__(self, items):
+        self._items = items
+
+    def __getitem__(self, item):
+        _, _, value = self._data[item]
```

Comment:
Where does `self._data` come from here?

Reply:
Good catch, thanks - should be `self._items`.
Missing test coverage for this method, which needs filling in.